### ISSUE TITLE
Inverse synctex with internal pdf viewer using VS Code is not working in 9.20.0

### ISSUE THREAD

**Issue by JanLoskot** [James-Yu/LaTeX-Workshop#4244]
Hello,

It seems like in the newest version 9.20.0, the inverse synctex has stopped working. I checked and in the previous version (9.19.2) both the forward and inverse synctex are working properly. It may have something to do with changed highlight style of the forward synctex (A "flash" effect in the place of cursor in the code was used previously, now the forward synctex hightlights the whole sentence). 
I didn't change any settings, I only updated the workshop.

TL:DR: Forward and inverse synctex were both working in 9.19.2. Now in 9.20.0 only forward synctex is working, inverse is not. I am using the internal pdf viewer of latex workshop.

I am sorry if I'm missing something.

**Comment by James-Yu** [comment]
#4239
